src/seedu/addressbook/AddressBook.java

```diff
@@ -90,20 +92,31 @@
     private static final String MESSAGE_STORAGE_FILE_CREATED = "Created new empty storage file: %1$s";
     private static final String MESSAGE_WELCOME = "Welcome to your Address Book!";
     private static final String MESSAGE_USING_DEFAULT_FILE = "Using default storage file : " + DEFAULT_STORAGE_FILEPATH;
+    private static final String MESSAGE_TO_EDIT_PERSON_NAME = "Please Key in New Name, leave blank to keep the same";
```

Comment:
notice how some of the variables are grouped logically in the given code;
you could consider doing that with the new variables added as well